kepler/tasks.py

```diff
@@ -76,12 +78,33 @@ def submit_to_dspace(job, data):
         pkg.metadata = _fgdc_to_mods(get_fgdc(data))
         with tempfile.NamedTemporaryFile(suffix='.zip') as fp:
             pkg.write(fp)
-            handle = sword.submit(current_app.config['SWORD_SERVICE_URL'], fp.name,
-                                  auth=(username, password))
+            handle = sword.submit(current_app.config['SWORD_SERVICE_URL'],
+                                  fp.name, auth=(username, password))
         job.item.handle = handle
         db.session.commit()
 
 
+def get_geotiff_url_from_dspace(job):
+    """Retrieve the GeoTIFF URL from a DSpace Handle.
+
+        .. note:: assumes the OAI-ORE only contains a single TIFF.
+
+        :param job: :class:`~kepler.models.Job`
+    """
+    handle = job.item.handle.replace('http://hdl.handle.net/', '')
+    ore_url = current_app.config['OAI_ORE_URL'] + handle + '/ore.xml'
+    r = requests.get(ore_url)
+    r.raise_for_status()
+    doc = ET.fromstring(r.text)
+    image_urls = doc.findall(
```

Comment:
I seem to recall a conversation about this where it was decided we'd just support one .tif ;)

So... we want to freak out for 0 or more than 1 .tif returned?

Also, that xpath statement is pretty slick. As you may have noticed, I'm more of a "get it close enough and deal with it in the app". This is gonna make me up my game. :+1: